### PR TITLE
new Plugin "Project Invitation"

### PR DESCRIPTION
**Pull request by Hussein-Khalil**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -946,5 +946,17 @@
...
+    },
+    "ProjectInvitation": {
+        "title": "Project Invitation",
+        "version": "1.0.0",
+        "author": "Andrei Volgin / Hussein Khalil, ipunkt Business Solutions",
+        "license": "MIT",
+        "description": "Plugin is used to invite new users to actual Project by typing user's email. Input for invitation is located in Project Settings->Permissions.",
+        "homepage": "https://github.com/ipunkt/KanboardProjectInvitation",
+        "readme": "https://github.com/ipunkt/KanboardProjectInvitation/blob/master/README.md",
+        "download": "https://github.com/ipunkt/KanboardProjectInvitation/releases/download/v1.0.0/KanboardProjectInvitation-1.0.0.zip",
```
Comment (fguillot) [inline plugins.json:958]: The folder name must match the plugin namespace.

```bash
$ unzip KanboardProjectInvitation-1.0.0.zip 
Archive:  KanboardProjectInvitation-1.0.0.zip
56a19c7ea67e67874e16ce9e59f099f8f498045e
   creating: KanboardProjectInvitation/
   creating: KanboardProjectInvitation/Controller/
  inflating: KanboardProjectInvitation/Controller/ProjectInviteController.php  
   creating: KanboardProjectInvitation/Data/
  inflating: KanboardProjectInvitation/Data/config.php  
  inflating: KanboardProjectInvitation/LICENSE  
   creating: KanboardProjectInvitation/Locale/
   creating: KanboardProjectInvitation/Locale/de_DE/
  inflating: KanboardProjectInvitation/Locale/de_DE/translations.php  
   creating: KanboardProjectInvitation/Locale/ru_RU/
  inflating: KanboardProjectInvitation/Locale/ru_RU/translations.php  
  inflating: KanboardProjectInvitation/Plugin.php  
  inflating: KanboardProjectInvitation/README.md  
   creating: KanboardProjectInvitation/Template/
  inflating: KanboardProjectInvitation/Template/users-override.php  
  inflating: KanboardProjectInvitation/Template/users.php  
   creating: KanboardProjectInvitation/docker/
  inflating: KanboardProjectInvitation/docker-compose.yml  
   creating: KanboardProjectInvitation/docker/conf.d/
  inflating: KanboardProjectInvitation/docker/conf.d/local.ini
```

```php
<?php

namespace Kanboard\Plugin\ProjectInvitation;

use Kanboard\Core\Plugin\Base;
```

**Review by fguillot — CHANGES_REQUESTED** [review]

File: plugins.json (hunk 1 above)
Comment (Hussein-Khalil) [reply plugins.json]: @fguillot We have changed the namespace in Plugin. Everything in website request stays the same.

`<?php

namespace Kanboard\Plugin\KanboardProjectInvitation;

use Kanboard\Core\Plugin\Base;`

Comment (Hussein-Khalil) [reply plugins.json:958]: @fguillot We have changed the `Namespace` in our Plugin as well as in Website.

`namespace Kanboard\Plugin\KanboardProjectInvitation;`

File: plugins.json [hunk 2]
```diff
@@ -931,7 +931,7 @@
         "description": "This plugin gives the ability to assign actual Task (direct in comment-create section) to one of the users in Project.",
         "homepage": "https://github.com/ipunkt/KanboardCommentActions",
         "readme": "https://github.com/ipunkt/KanboardCommentActions/blob/master/README.md",
-        "download": "https://github.com/ipunkt/KanboardCommentActions/releases/download/1.0.0/KanboardCommentActions-1.0.0.zip",
+        "download": "https://github.com/ipunkt/KanboardCommentActions/releases/download/v1.0.0/KanboardCommentActions-1.0.0.zip",
```
Comment (fguillot) [inline plugins.json:934]: This link is now returning a 404:

```
wget https://github.com/ipunkt/KanboardCommentActions/releases/download/v1.0.0/KanboardCommentActions-1.0.0.zip
--2019-10-28 18:59:36--  https://github.com/ipunkt/KanboardCommentActions/releases/download/v1.0.0/KanboardCommentActions-1.0.0.zip
Resolving github.com (github.com)... 192.30.253.113
Connecting to github.com (github.com)|192.30.253.113|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2019-10-28 18:59:37 ERROR 404: Not Found.
```

Comment (Hussein-Khalil) [reply plugins.json:934]: @fguillot i have tested the link and functional [download](https://github.com/ipunkt/KanboardCommentActions/releases/download/v1.0.0/KanboardCommentActions-1.0.0.zip )
could you please accept the pull request.

**Pull request merged**